pkg/cli/image/mirror/mirror.go

```diff
@@ -179,6 +185,14 @@ func NewCmdMirrorImage(streams genericclioptions.IOStreams) *cobra.Command {
 }
 
 func (o *MirrorImageOptions) Complete(cmd *cobra.Command, args []string) error {
+	if o.KeepManifestList && len(o.FilterOptions.FilterByOS) > 0 && !o.FilterOptions.IsWildcardFilter() {
```

Comment:
@soltysh wdyt about this?  add this line to Complete like so, rather than in a new Validate fn, then keep the current Validate fn as/is (wrapper for FilterOptions.Validate), then no other change necessary, and Complete then Validate order is restored.

Reply:
ok, I added this line to the Validate Fn, as it should be, and also _only_ default filter to `.*` if user has not explicitly set that flag. I think this is good now, thanks for your suggestions/review